### PR TITLE
Add webhook example to Cloud SOAR documentation

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds a webhook example to the Cloud SOAR documentation here:
https://help.sumologic.com/docs/cloud-soar/cloud-soar-integration-framework/#example-files

Issue number: GitHub issue #2833 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cloud-soar/cloud-soar-integration-framework.md [hunk 1]
```diff
@@ -913,6 +913,69 @@ output:
...
+"hours": 2,
+"percentage": 90,
+"priority": 88,
+"report_time": "2023-01-19T15:28:38.000Z",
+"start_time": "2023-01-19T15:28:34.000Z",
+"end_time": "2023-01-19T15:28:37.000Z",
+"status": 776,
+"assigned_to": 3,
+"additional_info": "lorem ipsum additional info text"
+}
```
Comment (kimsauce) [inline docs/cloud-soar/cloud-soar-integration-framework.md:966]: indentation

```suggestion
{
  "title": "test",
  "hours": 2,
  "percentage": 90,
  "priority": 88,
  "report_time": "2023-01-19T15:28:38.000Z",
  "start_time": "2023-01-19T15:28:34.000Z",
  "end_time": "2023-01-19T15:28:37.000Z",
  "status": 776,
  "assigned_to": 3,
  "additional_info": "lorem ipsum additional info text"
} 
```

Comment (kimsauce) [inline docs/cloud-soar/cloud-soar-integration-framework.md:966]: indentation

```suggestion
{
  "title": "test",
  "hours": 2,
  "percentage": 90,
  "priority": 88,
  "report_time": "2023-01-19T15:28:38.000Z",
  "start_time": "2023-01-19T15:28:34.000Z",
  "end_time": "2023-01-19T15:28:37.000Z",
  "status": 776,
  "assigned_to": 3,
  "additional_info": "lorem ipsum additional info text"
} 
```

File: docs/cloud-soar/cloud-soar-integration-framework.md [hunk 2]
```diff
@@ -913,6 +913,69 @@ output:
...
+    print(payload.get("title"))
+fields:
+  - id: payload
+    label: 'payload'
+    type: text      
+    required: true
+```
+
+Now you can perform a `POST` request to the [Cloud SOAR API](/docs/cloud-soar/cloud-soar-apis/) `/webhook` resource with a raw payload:
+```
```
Comment (kimsauce) [inline docs/cloud-soar/cloud-soar-integration-framework.md:954]:
```suggestion
```json
```

File: docs/cloud-soar/cloud-soar-integration-framework.md [hunk 3]
```diff
@@ -913,6 +913,69 @@ output:
     - path : 'exit_condition'
 ```
 
+### Trigger webhook definition file
+
+```
```
Comment (kimsauce) [inline docs/cloud-soar/cloud-soar-integration-framework.md:918]:
```suggestion
```yaml
```

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce!

**Review by kimsauce — APPROVED** [review]

**Pull request merged**